### PR TITLE
Almost complete two issues i cannot resolve

### PR DESCRIPTION
**Pull request by Hoobachube**
1st draft almost complete. I have used my current c.v as the template.  Obviously I will edit it to a more useful c.v to do with coding when the time arises I cannot seem to format this so there must be some error that vscode cannot deal with but I cant find it.  Also I have Bullet points that should not be there in the cover supervisor section that i cant remove.

### CODE REVIEW

File: cv.html [hunk 1]
```diff
@@ -38,7 +38,125 @@
...
+      <h1 style="text-align: center;">Jack Dachtler</h1>
+  <h2 style="text-align: center;">Production Manager / Supervisor</h2>
+  </div>
+  <i>dachtler@live.co.uk 07754 388 622  Dartford, UK</i> 
+  
+  <div class="container pt-3">
+  <p>
+  A tactical and results-focused production manager with outstanding project and people management skills. Organised, process-driven with the knowledge to develop efficient procedures, in a production environment. Develop and implement strategy and effective new systems and processes. An influential communicator and motivational leader proven in developing cross-functional, successful teams to deliver a profitable production operation. Widely regarded as a strong and supportive leader who actively promotes an environment of continual improvement and best practice.</P>
+  <p>
+    <b><i>Areas of Expertise include:</i></B>
```
Comment (alexw1983) [inline cv.html]: Avoid `<b>` and `<i>` tags if a more descriptive tag is available

File: cv.html [hunk 2]
```diff
@@ -38,7 +38,125 @@
...
+  </p>
+  
+  <table class="table table-borderless">
```
Comment (alexw1983) [inline cv.html]: Three columns can confuse people stick to one column for articles.

File: cv.html [hunk 3]
```diff
@@ -38,7 +38,125 @@
...
+  <td>Project Management</td>
+  <td>Problem Solving</td>
+  </tr>
+  </tbody>
+  </table>
+  
+  
+    <br></br>
+  
+  <h3 style="text-align: center;">FERNDALE FOODS</h3>
```
Comment (alexw1983) [inline cv.html]: don't use inline styles

File: cv.html [hunk 4]
```diff
@@ -38,7 +38,125 @@
...
+  <p style="text-align: center;">
+  2015 to Present <i>A food and beverage producer.</i>
+  </p>
+  
+  <h4 style="text-align: center;">NIGHT SUPERVISOR</h4>
+  <p>
+  Lead a high performing team. Prioritise and share tasks. Make decisions to plan and deliver a highly successful production operation. Collaborate with internal departments, customers, and key stakeholders to agree and define operational requirements, in accordance with overall business strategy. Forecast requirements for production schedule development. Devise and issue reports, including waste management, profit & loss, and stock auditing. Execute risk assessments. Troubleshoot and resolve problems. Store and handle equipment and materials in line with safety and security requirements. Enforce and adhere to hygiene, health and safety rules and regulations.
+  </p>
+  <b>Key Accomplishments:</b>
```
Comment (alexw1983) [inline cv.html]: is this a heading or a bold bit of text

**Review by alexw1983 — CHANGES_REQUESTED** [review]

**Comment by Hoobachube** [comment]
I must be trying to implement the style sheet incorrectly.  I tried the remove text-align attribute to a style sheet but didn't work

**Comment by Hoobachube** [comment]
I have added the blog style in the about me page.  I aim to remove the c.v page completely but wanted to leave it for the moment to help me hone my css skills before removing.  Hope this was alright.   I haven't had time to add much in the way of styling yet.  Its mainly just the 1st draft of raw text.

**Pull request closed without merging**